terraform/service_server.tf

```diff
@@ -80,6 +80,12 @@ resource "google_project_iam_member" "server-observability" {
   member  = "serviceAccount:${google_service_account.server.email}"
 }
 
+resource "google_kms_key_ring_iam_member" "server-verification-key-admin" {
+  key_ring_id = google_kms_key_ring.verification.self_link
+  role        = "roles/roles/cloudkms.admin"
```

Comment:
I forget if admin includes signer/verifier. If it does, we should drop the old permission.

Reply:
doesn't seem to -> https://cloud.google.com/kms/docs/reference/permissions-and-roles#predefined